src/db/connection.ts

```diff
@@ -0,0 +1,13 @@
+import { Sequelize } from 'sequelize';
+import { PASS, NAME, DATABASE } from '../config';
+
+const sequelize = new Sequelize({
```

Comment:
as we're going to have a database URL for production, So I'd prefer to not specify every single attribute of db connection, Instead, we can provide `Sequelize` with database connection URL as follows,

Reply:
please set logging to `false` as it's a bit annoying to log every db operation